### DISCUSSION TITLE
Failed to start jupyter in the environment python 3.11.2?

### DISCUSSION THREAD

**Discussion by tommieeeeeee** [AammarTufail/python_ka_chilla2023#110] [Q&A]
![Error](https://user-images.githubusercontent.com/126323501/221352981-6a5693ee-c9ca-447f-b557-0bfec342a324.PNG)
iska koi asaan tareqa bataye smjh nahi arhi ... python and vs ko 2 dafa reinstall kia same issue

**Comment by Muhammad-Ali-Butt** [comment]
[SOLUTION](https://stackoverflow.com/questions/57983475/jupyter-server-crashed-unable-to-connect-error-code-from-jupyter-1)
Try this:
In VS Code you can go to command palette by ctrl+shift+P and type jupyter and you will get a dropdown and select filter kernels it should activate your jupyter sessions.


If it does not work then check out the above link and read the top answer hopefully your error will be fixed with it but if it persists then hit me up here again i will gladly help :)
